docs/insights/index.md

```diff
@@ -0,0 +1,88 @@
+- Help introduce change with data and collaboration to make a business case
+- Share successes and learn from failures to continuously improve
+
+## Understand performance of your projects with Project level Insights
+
+![Project Insights Deployment Frequency](images/project.png)
+
+Project level insights are available as a new tab in every project so you can understand performance of your projects across Channels, Environments and Tenants. Each metric can be seen at a summary level and insights can also be filtered to time frames including last month, quarter and year, channels and environments, as well as being exported into CSV.
+
+Project level insights are available to all customers out-of-the-box meaning you don’t have to buy or subscribe to another tool. If you’re already a user, Octopus already has all the data it needs to help you uncover rich insights based on your deployment history.
+
+## Gain insight across projects and teams with the Space level insights
+
+This release also includes additional insights capabilities for Enterprise customers. For customers at larger companies, we have built additional capabilities that make it easier to gain insight using DORA metrics in larger multi-team, multi-site and multi-project scenarios.
+
+Space level insights are available via the Insights tab and provide actionable DORA metrics for more complex scenarios across projects, projects groups, environments, or tenants. This enables managers and decision makers to get far more insight into the DevOps performance of their organization in line with their business context, such as team, portfolio or platform. Space level insights:
+
+- Aggregate data across your space so you can compare and contrast metrics across projects to identify what is working and what isn’t
+- Inform better decision making: identify problems, track improvements and celebrate successes 
+- Help you quantify DevOps performance based on what is actually happening as shown by data
+
+With the Space level insights, you can build reports with the relevant data that matters to you, choosing only relevant projects, project groups, environments, tenants. View information at a high level or drill down in project specific data over time.
+
+This also allows you to easily identify contributors to a performance trend you see at the aggregated level. It provides a breakdown of highest and lowest performing projects, and releases that may have a significant impact on the overall performance. This enables you to identify where things are improving or declining and take action based on that.
+
+> Users who have view permissions to Space-level Insights reports will see sanitized data on projects / environments they don’t have access to.
+
+Enterprise DevOps Insights (Space-level) are available as a part of the Octopus Deploy Enterprise tier.
+
+Contact our sales team for a demo and learn more about [Octopus Deploy pricing](https://octopus.com/pricing)
```

Comment:
I'm unsure if there is a past precedent for adding a request for customers to contact sales for a demo or pricing links in the docs. That doesn't mean we can't though :smile:.

```suggestion
Space level DevOps insights are available to customers with an [enterprise license](https://octopus.com/enterprise).
```